recipes/textnets/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "textnets" %}
+{% set version = "0.4.10" %}
+{% set hash = "c9c917a76ee02fe34ee4dbf41428a1df26a07430234f578f45dc9e8a8cb13c79" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - textnets = textnets.cli:main
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - Click >=7.0
```

conda-forge packages are always lowercase
```suggestion
    - click >=7.0
```